### DISCUSSION TITLE
Börsenpreisgesteuertes Einschalten von Geräten?

### DISCUSSION THREAD

**Discussion by butterwecksolutions** [camueller/SmartApplianceEnabler#437] [Allgemein]
Hallo zusammen, könnte man nicht eine börsenpreisgesteuerte Schaltfunktion einbauen wie ich es hier umgesetzt habe?
https://github.com/christian1980nrw/Victron-ESS__AVM-Fritz-DECT200-210__Spotmarket-Switcher
Bei Interesse könnt ihr meinen Code gerne übernehmen @camueller .

Viele Grüße!
Christian

**Comment by camueller** [comment]
Ich habe schon einige Mal in diese Richtung gedacht. Insbesondere, nachdem einer der ersten User des SAE deswegen vom SHM und vom SAE weggegangen ist.

**ABER: Die Berücksichtigung von Strompreisen ist Aufgabe des Enery Management Systems und das ist der SHM und nicht der SAE.** Ich hoffe, dass der SHM mittelfristig variable Strompreise unterstützt.

Vor einiger Zeit hatte ich mal geschaut, ob sich die Nutzung solcher Tarife für mich persönlich rechnet, aber die Einsparungen waren marginal. Insofern habe ich auch von dieser Seite keine Nachteile aus der Tatsache, dass der SHM das aktuell nicht kann.

**Reply by butterwecksolutions** [reply]
Danke für Deine schnelle Antwort.
Vermutlich liegt die bisherige negative Berechnung daran, dass die Versorger mit Festpreisen erst mit Zeitverzögerung auf die Marktentwicklung reagieren. Entweder in Form einer Insolvenz oder durch deutliche und langfristige Preiserhöhungen. Naja, wir werden sehen wie die weitere Entwicklung ist, aber in Anbetracht der kommenden Rezession gehe ich von einem Überangebot an Strom und somit gemäßigten Börsenkursen aus. 
Ihr könnt es ja mal im Hinterkopf behalten, das es dafür eine funktionsfähige Lösung gibt, die Integration in Eure GUI als optionales Modul wäre doch bestimmt mit vertretbarem Aufwand machbar. Dann wären es halt zwei Quellen, SHM und Börse die sich gegenseitig ergänzen oder auch mal überlappen. Dadurch könnte das E-Auto auch im Winter oder nachts günstig laden.

Viele Grüße!
